src/components/Game.tsx

```diff
@@ -0,0 +1,169 @@
+import { useEffect, useReducer, useState } from 'react';
+import { calculateWinner, createArrayAttributes } from '../utilities/';
+import Board from './Board';
+import GameInfo from './GameInfo';
+import Move from './Move';
+import Footer from './Footer';
+
+/**
+ * Type for game state properties
+ */
+type State = {
```

Comment:
```suggestion
type GameState = {
```

I would more explicit with this name